index.bs

```diff
@@ -9815,6 +9815,24 @@ The method is invoked with the following arguments:
 	<a>computedNumberOfChannels</a> only when the node has single
 	output.
 
+	<div algorithm="check-process-outputs">
+		Let <var>outputs_in</var> be the <code>outputs</code> passed
+		to <code>process()</code>.  let <var>outputs_out</var> be the
+		<code>outputs</code> parameter on return from
+		<code>process()</code>.
+
+		1. Check <var>outputs_out</var>:
+			1. <var>outputs_out</var> must be a sequence whose length is the same as the length of <var>outputs_in</var>.
+			1. <var>outputs_out[n]</var> must be a sequence whose length is the same the length of <var>outputs_in[n]</var>.
+			1. <var>outputs_out[n][m]</var> must be a {{Float32Array}} whose length is the same the length of <var>outputs_in[n][m]</var>.
+		1. If all checks above pass, exit this algorithm.
+		1. Otherwise some check failed and perform the following steps:
+			1. Deactivate the processor, resulting in silent output from the processor for all subsequent renderings.
+			1. Throw an exception
+			1. Fire the {{AudioWorkletNode}}'s {{onprocessorerror}} event.
```

Comment:
This is best worded as "queue a task on some event loop to fire the `"processorerror"` event".

Reply:
Done.  Is this good enough for now?